README.md

```diff
@@ -2,64 +2,53 @@
 
 Template to start from before starting frontend projects.
 
-## Install template
-
-- Create empty repository for your project. [learn how](#) _need a link here_
-- Clone boilerplate into project directory and navigate there.
-```bash
-$ git clone https://github.com/mate-academy/gulp-template.git project-name
-$ cd project-name
-```
-- Change remote repository to recently created
-```bash
-$ git remote set-url origin github-link-of-project
-```
-- Install dependencies
-```bash
-$ npm install
-```
-
-All packages will be installed after executing the script above.
+## Task
 
-## Development mode 
-
-```bash
-$ npm run start
-```
+You need to create a function that generates calendar for a given pair 
+(year, month).
+The calendar must be a table with `thead` and `tbody`, where day it's `TD`.
 
-## Build the project
+Function `calendarTable(year, month)`
 
-```bash
-$ npm run build
-```
+## Workflow
 
-## Linters
+- Fork the repository with task
+- Clone forked repository 
+    ```bash
+    git clone git@github.com:<user_name>>/<task_repository>.git
+    ```
+- Run `npm install` to install dependencies.
+- Then `Develop`
```

Comment:
Agree with @danmysak that this line is not super informative.

Reply:
removed